### PR TITLE
Added CMIS list documents connector

### PR DESCRIPTION
**Pull request by pozil**

### CODE REVIEW

**Comment by educhastenier** [comment]
Is there a corresponding pull request to include this new connector in default studio connectors, so that we merge the 2 PRs at the same time?

**Comment by pozil** [comment]
I guess not. Sorry I am not aware of the procedure/project to include it in default studio connectors but that is my purpose.

**Comment by educhastenier** [comment]
Studio developers can help you on that.

File: bonita-connector-cmis-listdocuments-impl/pom.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <artifactId>bonita-connector-cmis</artifactId>
+        <groupId>org.bonitasoft.connectors</groupId>
+        <version>1.0.0</version>
+    </parent>
+    
+    <artifactId>bonita-connector-cmis-listdocuments-impl</artifactId>
+	<version>2.0.1</version>
+	
+    <properties>
+        <definition.version>2.0.1</definition.version>
```
Comment (rbioteau) [inline bonita-connector-cmis-listdocuments-impl/pom.xml]: use 1.0.0

File: bonita-connector-cmis-listdocuments-impl/pom.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>bonita-connector-cmis</artifactId>
+        <groupId>org.bonitasoft.connectors</groupId>
+        <version>1.0.0</version>
+    </parent>
+    
+    <artifactId>bonita-connector-cmis-listdocuments-impl</artifactId>
+	<version>2.0.1</version>
```
Comment (rbioteau) [inline bonita-connector-cmis-listdocuments-impl/pom.xml]: use 1.0.0

File: bonita-connector-cmis-listdocuments-def/pom.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>bonita-connector-cmis</artifactId>
+        <groupId>org.bonitasoft.connectors</groupId>
+        <version>1.0.0</version>
+    </parent>
+    
+
+    <artifactId>bonita-connector-cmis-listdocuments-def</artifactId>
+  	<version>2.0.1</version>
```
Comment (rbioteau) [inline bonita-connector-cmis-listdocuments-def/pom.xml]: Use version 1.0.0

**Comment by pozil** [comment]
Requested changes applied.

**Pull request merged**